index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title></title>
+	<link rel="stylesheet" type="text/css" href="css/style.css">
+	<link rel="stylesheet" type="text/css" href="css/sweetalert.css">
+	<script type="text/javascript" src = "js/sweetalert.min.js"></script>
+</head>
+<body>
+<div id="header" class="page-header">
+  <h1>Pull up a chair! <small>Come see the pictures we've got!</small></h1>
+</div>
+<div class='container'>
+<div class="row">
+<div class="col-md-6 col-md-offset-3">
+<form id="search-form" class="form-inline">
+  <div class="form-group">
+    <input type="text" class="form-control input-lg" id="searchFor" placeholder="What do you want to see?">
+  </div>
+  <button type="submit" class="btn btn-success btn-lg active">Search r/Hearthstone</button>
+  <button type="submit" class="btn btn-danger btn-lg active">Stop Search</button>
+</form>
+</div>
+</div>
+<div class="row">
+<div id="picContain" class="col-md-8 col-md-offset-2">
+<img id="pic1" class="img-responsive" src="#">
```

Comment:
Fix your indentation. You can't give images # as a placeholder like you can give links that. You wind up with a broken image. The business of the background distracts you from seeing the broken image, but it's there.